File: CS315-HW2/Codes/altiparmak_gokberk.rb
# Conditional exit
for i in 1..3 do
    puts "*\n"
end

# Unconditional exit(break)
x = 4
loop do
    if x == 5
        break
    end
    puts "****\n"
    x = x + 1
end

# Unconditional exit(continue)
loop do
    x = x + 1
    if x < 10
        next
    end
    puts "*********\n"
    break
end

# Enclosed loop exit
for i in 0..2 do
    for j in 0..1 do
      if i == 1 
        break
      end
      puts "j = #{j}"
    end
    puts "i = #{i}"
  end

File: CS315-HW2/Codes/altiparmak_gokberk.rs
fn main() {
    // Conditional exit
    for i in 1..3 {
      println!("{:?}", i);
    }

    // Unconditional exit(break)
    let mut x = 4;
    loop {
        if x == 6 {
            break;
        }
        x += 1;
        println!("***");
    }

    // Unconditional exit(continue)
    loop {
        x += 1;
        if x < 10 {
            continue;
        }
        println!("******");
        break;
    }

    // Enclosed loops exit
    for i in 0..3 {
        for j in 0..2 {
          if i == 1 {
            break;
          }
          println!("j = {}", j);
        }
        println!("i = {}", i);
      }
}